flocker/node/_deploy.py

```diff
@@ -473,6 +475,7 @@ def applications_from_units(result):
                 not_running=not_running,
                 used_ports=self.network.enumerate_used_ports(),
                 manifestations=manifestations,
+                paths=manifestation_paths,
```

Comment:
I forget why this isn't represented using a new `path` attribute on `Manifestation`.  Can you write that down somewhere?